homework/week1/2025320058-twuel0921/Main.java

```diff
@@ -0,0 +1,44 @@
+import java.util.*;
+
+public class Main {
+    private static int[] arr;
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int n = sc.nextInt();
+        int m = sc.nextInt();
+        arr = new int[n];
+        for(int i = 0; i < n; i++) arr[i] = sc.nextInt();
+        Arrays.sort(arr);
+        for(int i = 0; i < n; i++) {
+            int[] sub = new int[m];
+            Arrays.fill(sub, 0);
+            boolean[] visited = new boolean[n];
+            Arrays.fill(visited, false);
+            sub[0] = arr[i];
+            visited[i] = true;
+            dfs(sub, visited, 1);
```

Comment:
아시겠지만 1 대신 0을 넣고 호출해서 맞으려면, for문을 없애야합니다.
```suggestion
        int[] sub = new int[m];
        Arrays.fill(sub, 0);
        boolean[] visited = new boolean[n];
        Arrays.fill(visited, false);
        dfs(sub, visited, 0);
```